### DISCUSSION TITLE
App Setting images do not load

### DISCUSSION THREAD

**Discussion by mattpolanieckidev** [mozilla/hubs#4764] [Hubs Cloud Questions]
Hello,

We have uploaded images under App Settings / Images but they do not appear. What could be something to check? 

![image](https://user-images.githubusercontent.com/10778873/138290282-45a426d3-dd0e-4836-9c28-9018ea7038d9.png)

![image](https://user-images.githubusercontent.com/10778873/138290449-38c4d140-48b8-44be-8b3d-d9e02f118cd1.png)

**Comment by pattersonbl2** [comment]
Is it possible for you to send me these images so I can try to upload them into my hub's cloud instance?

**Reply by mattpolanieckidev** [reply]
Hi, we tried with any random jpeg or png. Nothing will display.

**Reply by Softvision-GeluHaiduc** [reply]
@mattpolanieckidev maybe this has something to do with this issue, https://github.com/mozilla/hubs-cloud/issues/175#issuecomment-948006230, Robin has some pointers on where to look in the console to check

**Reply by mattpolanieckidev** [reply]
I'm not able to find anything. Looking at the page source, it's not even looking like the server is requesting an image from an unavailable repository. Any other suggestions? 

![image](https://user-images.githubusercontent.com/10778873/139118776-13f58027-254c-4ca6-8018-d47a27d3e112.png)

**Reply by mattpolanieckidev** [reply]
Additionally, we see in the JSON that the image urls are being encoded incorrectly. 

![image](https://user-images.githubusercontent.com/10778873/139123995-ce677625-95c1-46fd-be63-a3fac15b4c4a.png)

**Reply by pattersonbl2** [reply]
Could you possibly send the images to me so I can upload them to my instance ?
